.github/workflows/release.yml

```diff
@@ -6,35 +6,21 @@ on:
     types: [prereleased, released]
 
 env:
-  ASSET_NAME: skeleton-aws-lambda.zip
   PIP_CACHE_DIR: ~/.cache/pip
-
 jobs:
   release:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        # Versions supported by AWS and lambci/lambda images
+        python-version: [3.6, 3.7, 3.8]
     steps:
       - uses: actions/checkout@v2
-      - id: setup-python
-        uses: actions/setup-python@v2
-        with:
-          python-version: 3.8
-      - name: Cache lambda building environments
-        uses: actions/cache@v2
-        with:
-          path: ${{ env.PIP_CACHE_DIR }}
-          key: "release-${{ runner.os }}-\
-            py${{ steps.setup-python.outputs.python-version }}-\
-            ${{ hashFiles('**/requirements.txt') }}"
-          restore-keys: |
-            release-${{ runner.os }}-\
-            py${{ steps.setup-python.outputs.python-version }}-
-            release-${{ runner.os }}-
-      - name: Install dependencies
+      - name: Build Docker image
         run: |
-          python -m pip install --upgrade pip wheel
-          pip install --upgrade --requirement requirements.txt
-      - name: Build environment
-        run: docker-compose build
+          DOCKER_BUILDKIT=1 docker-compose build \
```

Comment:
What does `DOCKER_BUILDKIT=1` do?

Reply:
This tells Docker to use BuildKit per https://docs.docker.com/develop/develop-images/build_enhancements/

Reply:
I think including that link as a comment would be good, particularly since this is a skeleton.

Reply:
Please see f525714 for a comment with the link and a brief explanation.